src/client/Microsoft.Identity.Client.Desktop/Microsoft.Identity.Client.Desktop.csproj

```diff
@@ -49,18 +49,19 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetDesktop)' or '$(TargetFramework)' == '$(TargetFrameworkNetCore)'">
-    
     <Compile Include="$(PathToMsalSources)\Platforms\Features\WamBroker\**\*.cs" LinkBase="Features/WAM" />
     <PackageReference Include="Microsoft.Windows.SDK.Contracts" Version="10.0.17763.1000" />
     <Reference Include="System.Windows.Forms" />
-
     <Compile Include="$(PathToMsalSources)\Platforms\Features\WebView2WebUi\**\*.cs" LinkBase="Features/WebView2WebUi" />
-    
+
     <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.705.50" />
+  </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetCore)'">
+    <Compile Include="$(PathToMsalSources)\Platforms\Features\WinFormsLegacyWebUi\**\*.cs" LinkBase="Features\WinFormsLegacyWebUi" />
```

Comment:
Put this in a separate item group because `WinFormsLegacyWebUi` is already added to NetDesktop target in Msal library.